apc_lock.c

```diff
@@ -346,3 +346,4 @@ PHP_APCU_API void apc_lock_destroy(apc_lock_t *lock) {
 }
 #endif
 
+#include "apc_mutex.c"
```

Comment:
I have no idea how to get this to link, else ...

Reply:
Adding the file in `config.m4` should be enough to link it. You have to rerun buildconf/phpize and configure for the changes to apply though.

Reply:
doh, my error was that I had the #ifdef without including apc.h / config.h, which means my implementation was never compiled.